src/app/components/dotted-menu/movie-card-menu.component.spec.ts

```diff
@@ -16,10 +17,27 @@ describe('DottedMenuComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(MovieCardMenuComponent);
     component = fixture.componentInstance;
+    component.movie = new Movie(1, "testTitle", "testTagline", 5.6, 1000,
```

Comment:
not related to tests, but you can use `new Movie({...})` notation, it's commonly used when a function has too many arguments. Using an object as an argument makes it easier to not worry about order of arguments or skipping some unneeded